documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+# Consent decision-making
+
+The Openverse maintainers have elected to adopt a decision-making model based on
+the
+[Sociocracy practice of "consent decision-making"](https://patterns.sociocracy30.org/consent-decision-making.html).
+We hope this will accomplish the following:
+
+- Bring expediency to decision-making by giving guidelines for how and what
+  types of feedback to give
+- Adopt more specific standards about what types of feedback must be addressed
+  before a proposal can be ratified for implementation
+- Give specificity in feedback expectations, both for who will participate in a
+  decision-making process and what responsibilities and tools they have for
+  participating
+- Reaffirm trust between contributors when evaluating proposals
+
+Historically, each of these have presented problems for the Openverse project as
+it has grown. Decision-making can drag on for months, proposal reviews get stuck
+in feedback cycles over issues that could be addressed later on, and there were
+few, if any, expectations around who would participate in a discussion or what
+the expectations of that participation was. Adopting the process described
+below, we hope, will address some of the underlying causes of these problems, in
+addition to the benefits described above.
```

Comment:
This is good background for the motivation to adopt this, of course, but in making the document as succinct as possible and "get to the point" as early as it can, would you consider removing it or adding it to some type of appendix towards the end? Maybe a section on history or motivation or something?

I think the focus of the document should be "here is how we communicate in Openverse" and provide the necessary background for participating as quickly as possible.

Reply:
Typically, sections on motivation are listed at the top, I personally find them awkward when "appendicised" but we can do that if necessary. I plan to add a Table of Contents to the document. My hope was that the "Process summary" section would be the jumping in point if someone doesn't care about any of the other details. Certainly that's where I would jump to if I was reading a similar document.

What do you think?

Reply:
The ToC should alleviate my concern 👍

Reply:
Just to clarify on this one, the table of contents is built into the website version of the document so I don't think it's worth duplicating in the text of the document itself. You can see the preview of the website version in the link in the PR description.